src/js/components/stock-movement-wizard/outbound/PackingPage.jsx

```diff
@@ -249,7 +249,7 @@ class PackingPage extends Component {
     const status = 'CHECKING';
     const payload = { status };
 
-    if (this.state.values.statusCode !== status) {
+    if (this.state.values.statusCode !== 'PACKED') {
```

Comment:
How about case when we have no pack page?

Reply:
@awalkowiak fixed